### PR TITLE
Create home page

### PR DESCRIPTION
**Pull request by keebot-png**
Created a new GitHub repo.
 Added a .gitignore file.
 Set up [linters for HTML and CSS].
 Made sure that any files that are not relevant are ignored by git.
 Added a descriptive README file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (RolandM99) [inline .gitignore]: - [ ] Kindly, make sure that you add your `test.md` file in your `,gitignore` file to ignore it in your remote repository.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,20 @@
...
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- html
+- css
+- git
```
Comment (RolandM99) [inline README.md]: - [ ]  Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @keebot-png, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly, add some meaningful description in your PR. your PR should have descriptive text talking about the changes you made in your codes.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,56 +1,29 @@
...
+- css
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **keebot-png**
```
Comment (codecaiine) [inline README.md]: - [ ]  Good job so far your readme looks great :+1:  But as the reviewer mentioned make it more professional by adding some information about the author. Please bring back this section and replace the different handles and links with your own pieces of information in order to make it more professional :+1:

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+./test.md
```
Comment (codecaiine) [inline .gitignore]: - [ ] Your test.md is not ignored, please fix it because it's part of the requirement :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @keebot-png , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linters errors :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
- Great project implementation :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md [hunk 5]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```
Comment (sumairq) [inline test.md]: - [ ] The test.md file should not be present in your remote repository. To remove it from your repository please refer to the this [stack overflow](https://stackoverflow.com/questions/2047465/how-do-i-delete-a-file-from-a-git-repository) thread.

**Review by sumairq — CHANGES_REQUESTED** [review]
Hi @keebot-png  , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @keebot-png,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Removal of test file 

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**